web/session.py

```diff
@@ -347,7 +347,8 @@ def __getitem__(self, key):
             return self.decode(s.data)
 
     def __setitem__(self, key, value):
-        pickled = self.encode(value)
+        # Remove the leading `b` of bytes object.
```

Comment:
I think we can lose this comment because it is misleading to say that decoding bytes is merely removing the b from the front.

Reply:
The comment line was updated BEFORE your comment.
I  think it's necessary to leave some comment to help others who read the code to understand why we do `.encode(...).decode()` here. If the words are not good enough (English is not my first language), feel free to update it.